ftgo-order-service-api/src/main/java/net/chrisrichardson/ftgo/orderservice/api/web/ReviseOrderRequest.java

```diff
@@ -1,22 +1,22 @@
 package net.chrisrichardson.ftgo.orderservice.api.web;
 
-import java.util.Map;
+import net.chrisrichardson.ftgo.common.RevisedOrderLineItem;
 
 public class ReviseOrderRequest {
-  private Map<String, Integer> revisedLineItemQuantities;
+  private RevisedOrderLineItem[] revisedOrderLineItems;
```

Comment:
Isn't a `List<..>` more normal?

Reply:
Hm... I always use arrays in such cases, because comparing to generic collections arrays are strongly typed and always parsed correct. But probably list also can be parsed correctly by swagger, I will check.